stomp_moveit/src/cost_functions/collision_check.cpp

```diff
@@ -215,7 +205,8 @@ bool CollisionCheck::computeCosts(const Eigen::MatrixXd& parameters,
   costs = Eigen::VectorXd::Zero(num_timesteps);
 
   // resetting array
-  raw_costs_.setZero();
+  Eigen::VectorXd raw_costs = costs;
```

Comment:
Creating a new variable  here is more inefficient than preallocating it and setting it to zero

Reply:
This was required for parallel implementation of the code. It cannot change local members from multiple threads.